Yoast/wordpress-seo
# Use Estimated Reading Time indexable value in Enhanced Slack Integration

## Context


* The Enhanced Slack integration did its own calculations for the reading time. Now, we use the data that is stored on the indexables by the new analysis in premium.

## Summary


This PR can be summarized in the following changelog entry:

* Uses the Estimated Reading Time indexable value for the Enhanced Slack integration.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Open an old post, for which you are sure you have no reading time value in the database yet.
* Visit that post on the frontend, and view the headers (I always use `cmd + alt + u`).
* The Estimated Reading should not be visible ( e.g.:
```
<meta name="twitter:label2" content="Est. reading time">
<meta name="twitter:data2" content="16 minutes">
```
* Now edit the post. Verify in Redux that the Estimated Reading Time changed (or use `wp.data.select( "yoast-seo-premium/editor" ).getEstimatedReadingTime()` ).
* Save and update the post, verify that the above meta tags are now present on the frontend.
* Update your text (make it 3 times as long by copying, or if already long, delete half of it).
* Save and update, verify that the above meta tags have changed accordingly on the frontend.


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Go to a post on the frontend, and checkout the source.
* Verify that something like this is present in the header:
```
<meta name="twitter:label2" content="Est. reading time">
<meta name="twitter:data2" content="16 minutes">
```
* If not, go and edit the post. Save and update, and repeat the check on the frontend. It should now be present.
* Now edit the length of the content in the editor, by doubling or halving your post lenght.
* Update and save, and verify that on the frontend the reading time estimate has changed accordingly.


## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* Enhanced Slack sharing feature

## UI changes


## Documentation


## Quality assurance


P1-226
